testie.wren

```diff
@@ -25,13 +25,27 @@ class Testie {
         _skips = []
         _name = name
         _fails = 0
+        _abortAfterFailures = true
```

Comment:
Is this related to TAP somehow or just a general improvement?

Reply:
I want to be able to suppress the abort when there are failing tests: TAP specification does not allow for that output.

Reply:
The abort also triggers the error code (for the shell)... if you toggle this to false should we use `Process.exit(1)` or some such so that the failing status is still reported to the shell? Or does the TAP spec also cover this behavior?